src/data/STATUSES/SCH.js

```diff
@@ -1,4 +1,10 @@
 export default {
+	CLERIC_STANCE: {
```

Comment:
This should probably go in `ROLE.js`, with the other role actions.

I've not actually got any role actions in for SCH - it's not supported yet per se.

Reply:
Sure, I'll break out shared statuses. For clarification purposes, was it intended to mix statuses that afflict both players and enemies in the same file?

Reply:
Yeah. I'm only classifying actions/statuses by the job that actually uses them (or role, etc). Despite the different icon styles, XIV treats both buffs and debuffs as a homogeneous 'Status'.